react/ProductReference.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FunctionComponent, useState, useEffect } from 'react'
+import { Loading } from 'vtex.render-runtime'
+import { useCssHandles } from 'vtex.css-handles'
```

Comment:
Please fix the lint errors. 

On the root: `yarnl; yarn lint --fix`

Reply:
@kaisermann,

I corrected the Lint errors of my component, which is the ProductReference, but now the other Lint errors were not what I caused. This is something that came from VTEX. I kindly ask that you correct the remaining Lint problems. Because I will not correct mistakes that I did not cause.

Reply:
Sorry, I should have been clearer. I didn't mean for you to correct lint errors unrelated to your code 😉 Thanks